Session aware logout, backend logout url approach (#1876)
oauth2-proxy/oauth2-proxy: docs/docs/configuration/overview.md

| `--auth-logging-format` | string | Template for authentication log lines | see [Logging Configuration](#logging-configuration) |
| `--authenticated-emails-file` | string | authenticate against emails via file (one per line) | |
| `--azure-tenant` | string | go to a tenant-specific or common (tenant-independent) endpoint. | `"common"` |

> | `--backend-logout-url` | string | URL to call to perform backend logout, `{id_token}` would be replaced by the actual `id_token` if available in the session | |

Reviewer: ```suggestion
| `--backend-logout-url` | string | URL to call to perform backend logout, `{id_token}` in the url would be replaced by the actual `id_token` if available in the session | |
```

Author: rephrase suggested by tuunit includes this change

> | `--backend-logout-url` | string | URL to call to perform backend logout, `{id_token}` would be replaced by the actual `id_token` if available in the session | |

Reviewer 2: ```suggestion
| `--backend-logout-url` | string | URL to perform backend logout, if you use `{id_token}` in the url it will be replaced by the actual `id_token` of the user session | |
```
